pennylane_lightning/src/simulators/lightning_kokkos/bindings/LKokkosBindings.hpp

```diff
@@ -0,0 +1,226 @@
+ *
+ * @tparam StateVectorT
+ * @tparam PyClass
+ * @param pyclass Pybind11's measurements class to bind methods.
+ */
+template <class StateVectorT, class PyClass>
+void registerBackendSpecificMeasurements(PyClass &pyclass) {
+    using PrecisionT =
+        typename StateVectorT::PrecisionT; // Statevector's precision
+    using ComplexT =
+        typename StateVectorT::ComplexT; // Statevector's complex type
+    using ParamT = PrecisionT;           // Parameter's data precision
+
+    using np_arr_c = py::array_t<std::complex<ParamT>,
+                                 py::array::c_style | py::array::forcecast>;
+    using sparse_index_type = std::size_t;
+    using np_arr_sparse_ind =
+        py::array_t<sparse_index_type,
+                    py::array::c_style | py::array::forcecast>;
+
+    pyclass
+        .def("expval",
+             static_cast<PrecisionT (Measurements<StateVectorT>::*)(
+                 const std::string &, const std::vector<size_t> &)>(
+                 &Measurements<StateVectorT>::expval),
+             "Expected value of an operation by name.")
+        .def(
+            "expval",
+            [](Measurements<StateVectorT> &M, const np_arr_sparse_ind &row_map,
+               const np_arr_sparse_ind &entries, const np_arr_c &values) {
+                return M.expval(
+                    static_cast<sparse_index_type *>(row_map.request().ptr),
+                    static_cast<sparse_index_type>(row_map.request().size),
+                    static_cast<sparse_index_type *>(entries.request().ptr),
+                    static_cast<ComplexT *>(values.request().ptr),
+                    static_cast<sparse_index_type>(values.request().size));
+            },
+            "Expected value of a sparse Hamiltonian.")
+        .def("var",
+             [](Measurements<StateVectorT> &M, const std::string &operation,
+                const std::vector<size_t> &wires) {
+                 return M.var(operation, wires);
+             })
+        .def("var",
+             static_cast<PrecisionT (Measurements<StateVectorT>::*)(
+                 const std::string &, const std::vector<size_t> &)>(
+                 &Measurements<StateVectorT>::var),
+             "Variance of an operation by name.");
+    // pyclass.def(
+    //     "var",
+    //     [](Measurements<StateVectorT> &M, const np_arr_sparse_ind
+    //     &row_map,
+    //        const np_arr_sparse_ind &entries, const np_arr_c &values) {
+    //         return M.var(
+    //             static_cast<sparse_index_type *>(row_map.request().ptr),
+    //             static_cast<sparse_index_type>(row_map.request().size),
+    //             static_cast<sparse_index_type *>(entries.request().ptr),
+    //             static_cast<ComplexT *>(
+    //                 values.request().ptr),
+    //             static_cast<sparse_index_type>(values.request().size));
+    //     },
+    //     "Variance of a sparse Hamiltonian.");
```

Comment:
Will you still work on that, or the idea is to remove it?

Reply:
Added it.